bin-src/init.ts

```diff
@@ -0,0 +1,142 @@
+    const sbConfigPath = await getSBConfigFilePath()
+    const sbVersion = packageJson?.devDependencies?.storybook || packageJson?.dependencies?.storybook
+    if(sbConfigPath && sbVersion) {
+        installArgs = ['-D', 'chromatic',`chromatic-${testFramework}`, `@storybook/server-webpack5@${sbVersion}`]
+    }
+    const installCommand = await getPackageManagerInstallCommand(installArgs)
+    await execaCommand(installCommand)
+}
+
+const intializeChromatic = async ({testFramework, packageJson, packagePath}: {testFramework: TestFrameworkType, packageJson: PackageJson, packagePath: string }) => {
+    await addChromaticScriptToPackageJson({ packageJson, packagePath });
+    await createChromaticConfigFile({configFile: 'chromatic.config.json'})
+    switch (testFramework) {
+        case TestFramework.CYPRESS:
+            await installArchiveDependencies(packageJson, TestFramework.CYPRESS)
+            break;
+        case TestFramework.PLAYWRIGHT:
+            await installArchiveDependencies(packageJson, TestFramework.PLAYWRIGHT)
+            break;
+    
+        default:
+            break;
+    }
+}
+
+export async function main(argv: string[]) {
+    const { flags } = meow(
+        `
+        Usage
+          $ chromatic init [-f|--framework]
+
+        Options
+          --framework, -f <framework     Test Framework that you are aiming to use with Chromatic. (default: 'storybook')
+        `,
+        {
+            argv,
+            description: "Utility for setting up Chromatic",
+            flags: {
+                framework: {
+                    type: 'string',
+                    alias: 'f',
+                    default: TestFramework.STORYBOOK
+                }
+            }
+        }
+    )
+    console.log(
+        boxen('Welcome to Chromatic Initialization tool! This CLI will help get Chromatic setup within your project.', {
+            title: 'Chromatic Init',
+            titleAlignment: 'center',
+            textAlignment: 'center',
+            padding: 1,
+            borderStyle: 'double',
+            borderColor: '#FF4400',
+        })
+    )
+    
+    try {
+        const pkgInfo = await readPkgUp({ cwd: process.cwd() });
+        if (!pkgInfo) {
+            console.error(noPackageJson());
+            process.exit(253);
+        }
+
+        const { path: packagePath, packageJson } = pkgInfo;
+        const { testFramework } = await prompts([
+            {
+                type: flags.framework ? null : 'select',
+                name: 'testFramework',
+                message: 'What testing framework are you using?',
+                choices: [
+                    {title: 'Storybook', value: TestFramework.STORYBOOK},
+                    {title: 'Playwright', value: TestFramework.PLAYWRIGHT},
+                    {title: 'Cypress', value: TestFramework.CYPRESS},
+                ],
+                initial: 0
```

Comment:
@thafryer does this mean that if they don't pass the `-f` flag, we'll prompt them with this menu instead? That's pretty nice.

Reply:
Yes, that's correct!